conclave-host/src/main/kotlin/com/r3/conclave/host/internal/SocketHostEnclaveInterface.kt

```diff
@@ -21,10 +21,15 @@ import java.util.concurrent.Semaphore
  *  - Route calls from the enclave to the appropriate host side call handler, see [com.r3.conclave.common.internal.CallInterface]
  *  - Handle the low-level details of the messaging protocol (socket with streamed ECalls and OCalls).
  */
-class SocketHostEnclaveInterface(private val port: Int) : HostEnclaveInterface(), Closeable {
-    private lateinit var socket: Socket
+class SocketHostEnclaveInterface(port: Int = 0, private val maxConcurrentCalls: Int) : HostEnclaveInterface(), Closeable {
```

Comment:
Is actually defaulting to port 0 nice?

Reply:
Port zero means "system, please allocate me a free port"